tests/benchmarks/sys_kernel/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
   benchmark.kernel.core:
     arch_exclude: nios2 riscv32 xtensa
-    min_ram: 32
+    min_ram: 36
```

Comment:
I think this belongs in a separate commit in a separate PR.

Reply:
@henrikbrixandersen  this case is not buildable in twr_ke18f, add this will skip this case in board testing.

Reply:
> @henrikbrixandersen this case is not buildable in twr_ke18f, add this will skip this case in board testing.

I understand that. I still think it belongs in a separate commit with a dedicated commit log.

Reply:
I see, let me drop the PR, and create another one